welcome-message/banner.html

```diff
@@ -1,19 +1,21 @@
-<div data-o-component="o-expander" class="o-expander o-expander__content--expanded n-welcome-banner__wrapper" data-o-expander-shrink-to="hidden">
-	<h6 class="n-welcome-banner__heading n-welcome-banner__column n-welcome-banner__column--m">Welcome to the new FT.com.</h6>
-
-	<div class="o-expander__content n-welcome-banner__column n-welcome-banner__column--fill">
+<div class="n-welcome-banner__wrapper">
+	<h6 class="n-welcome-banner__heading n-welcome-banner__column">Welcome to the new FT.com.</h6>
```

Comment:
Layout issues:

In Chrome
![screen shot 2016-09-19 at 12 19 22](https://cloud.githubusercontent.com/assets/996106/18630241/075ba74a-7e63-11e6-9a7b-bc40e40aa830.png)

In IE
![ie](https://cloud.githubusercontent.com/assets/996106/18630296/7a529682-7e63-11e6-96fa-b719009b73be.png)